scanapp/migrations/0001_initial.py

```diff
@@ -20,108 +20,152 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Copyright',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
Please do NOT reformat or change auto-generated migrations

Reply:
Sure, let me try something to ignore migrations directory.

Reply:
Or improve the generator?

Reply:
@jayvdb this is the generator that is built-in Django and I am not sure it makes sense to spend any time to update this. Generated code should never be linted IMHO.